src/types/auth/otp.ts

```diff
@@ -0,0 +1,39 @@
+export interface TokenData {
+  token: string;
+  phoneNumber: string;
+  createdAt: string;
+}
+
+export type MFAMethod = "totp" | "backup" | "sms" | "email";
```

Comment:
```suggestion
export type MFAMethod = "totp" | "backup";
```

Our current backend support totp and backup, let's add remaining when we support them